web/docs/understand/language/pipelines.md

```diff
@@ -26,10 +26,33 @@ The diagram below illustrates the operator chaining:
 If a pipeline would not have a source and sink, it would "leak" data. We call
 pipelines that have both a source and sink a **closed pipeline**. VAST can only
 execute closed pipelines. A pipeline that solely consists of a chain
-transformations is an **open pipeline**. You can use open pipelines as building
-blocks to build nested pipelines:
-
-![Nested Pipeline](pipeline-nested.excalidraw.svg)
+transformations is an **open pipeline**.
+
+## Operator Overview
+
+Let's take an inside look at the anatomy of a pipeline: the operator building
+blocks. The diagram below shows the three major customization points of the
+pipeline execution engine. The operator SDK consists of transformations,
```

Comment:
```suggestion
pipeline execution engine. The operator SDK allows for building transformations,
```

Technically speaking the "SDK" would allow for building further. It's not the set of operators we ship ourselves.